tsl/src/bgw_policy/drop_chunks_api.c

```diff
@@ -37,6 +38,89 @@
 #define DEFAULT_RETRY_PERIOD                                                                       \
 	DatumGetIntervalP(DirectFunctionCall3(interval_in, CStringGetDatum("5 min"), InvalidOid, -1))
 
+typedef struct DropChunksMeta
+{
+	Hypertable *ht;
+	Oid ht_oid;
+	FormData_ts_interval *older_than;
+	bool is_matht; /* is this a materialized hypertable associated
+					  with a continuous aggregate */
+} DropChunksMeta;
+
+static void
+validate_drop_chunks_hypertable(Cache *hcache, Oid user_htoid, Oid older_than_type,
+								Datum older_than_datum, DropChunksMeta *meta)
+{
+	FormData_ts_interval *older_than;
+	ContinuousAggHypertableStatus status;
+
+	meta->ht = NULL;
+	meta->ht_oid = user_htoid;
+	meta->is_matht = false;
+	meta->ht = ts_hypertable_cache_get_entry(hcache, meta->ht_oid, true /* missing_ok */);
+	if (meta->ht != NULL)
+	{
+		if (meta->ht->fd.compressed)
+		{
+			ts_cache_release(hcache);
+			ereport(ERROR,
+					(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+					 errmsg("cannot add drop chunks policy to hypertable \"%s\" which contains "
```

Comment:
Suggestion: `"cannot add drop chunks policy to compressed hypertable \"%s\""`

(This also makes it consistent with the error below)